scripts/update_tests.sh

```
@@ -0,0 +1,302 @@
+if ! $check_skip_flag; then
+    echo "Updating Tests"
+
+    if [[ ${#libraries[@]} -eq 0 ]]; then
+        readarray -t libraries <<< $(find ${cpython_path} -type f -printf "%P\n" | grep -vE "$(IFS=\|; echo "${ignored_libraries[*]}")")
+    fi
+    update_tests "${libraries[@]}"
+else
+    echo "Checking Skips"
+
+    if [[ ${#libraries[@]} -eq 0 ]]; then
+        readarray -t libraries <<< $(find ${rpython_path} -iname "test_*.py" -type f -printf "%P\n" | grep -vE "$(IFS=\|; echo "${ignored_libraries[*]}")")
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)